src/main/java/frc/robot/commands/LockMode/Lock.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands.LockMode;
+
+import java.util.function.DoubleSupplier;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Camera.Camera;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+
+
+public class Lock extends CommandBase{
+    //Subsystem requirements for the command
+    public final void addRequirements(Camera camera){}
+    public final void addRequirements(DriveSubsystem drive){}
+
+    DriveSubsystem drive;
+    DoubleSupplier sideways;
+    DoubleSupplier forward;
+    boolean lockedOn = false;
+    Camera camera;
+
+    //PID controller for yawRate
+    PIDController yawRateController = new PIDController(1, 0, 0);
+
+    //This gets the controller inputs and drive information    
+    public Lock(DriveSubsystem drive, DoubleSupplier forward, DoubleSupplier sideways){
+        this.drive = drive;
+        this.forward = forward;
+        this.sideways = sideways;
+    }
+
+    public void cam(Camera camera){
+        this.camera = camera;
+        
+    }
+    
+    
+    
+    @Override
+    public void initialize(){}  
+
+    @Override
+    public boolean isFinished(){
+        return lockedOn;}
```

Comment:
We finish when the button is released / toggled, so this should return false always.

Reply:
I think we have different ideas for the design.  If one of the things our LED subsystem does is indicate whether the robot is locked on to the nearest target, this method should return whether the angle is within a certain range of zero.

Reply:
On second thought, that should probably be a seperate method.  We'll still need to change `lockedOn` based on the angle.

Reply:
`    @Override
    public boolean isFinished(){
        return false;
    }`